functions/gdl/language_controller.py

```diff
@@ -16,5 +16,6 @@ class LanguageController(Resource):
     @API.doc('List of available languages')
     @API.marshal_with(language, True)
     def get(self):
-        return [{'code': x['language'], 'description': ', '.join(tags.description(x['language']))} for x in
-                GDLConfig.GAMES_TABLE.scan()['Items']]
+        var languages = [item['language'] for item in GDLConfig.GAMES_TABLE.scan()['Items']]
+        var unique = list(dict.fromkeys(languages))
```

Comment:
Gjør denne det om til en dictionary for å fjerne duplikater også tilbake til en liste?

Reply:
Ja. Det er nettopp det den gjør. Eg fant ingen enkel måte å hente ut unike språk fra dynamodb uten å gjøre større endringer så dette er et slags hack. Dersom det er veldig mange spill kan vi potensiellt miste nokon språk så det bør sikkert kikkes på et alternativ.